integration_test/trackedusersreporting/tracked_users_reporting_test.go

```diff
@@ -292,6 +360,8 @@ func sendEvents(
 				return 0, fmt.Errorf("failed to send event to rudder server, status code: %d: %s", resp.StatusCode, string(b))
 			}
 			kithttputil.CloseResponse(resp)
+			// sleeping so that multiple requests can be sent to reporting
+			time.Sleep(time.Duration(rand.Intn(1000)) * time.Millisecond)
```

Comment:
Can we avoid this sleep in favour of another signal?

Reply:
Increased number of events and decreased processor batch size so that multiple processor batches will be processed.